### PR TITLE
fix: add default value font size

### PR DESCRIPTION
**Pull request by eliseeborn**
tested with various themes on V&A tenant, shows theme font size and color for all of them

### CODE REVIEW

File: src/ext/property-panel/settings.js [hunk 1]
```diff
@@ -28,7 +35,10 @@ const stylingPanel = {
                       width: 9,
                       min: 5,
                       max: 300,
-                      placeholder: '',
+                      defaultValue(item, data, args) {
+                        const currentTheme = args.theme.current();
+                        return stripPx(currentTheme.object?.straightTable?.header?.fontSize || currentTheme.fontSize);
```
Comment (cbt1) [inline src/ext/property-panel/settings.js:40]:
```suggestion
                        return parseInt(currentTheme.object?.straightTable?.header?.fontSize || currentTheme.fontSize, 10);
```

File: src/ext/property-panel/settings.js [hunk 2]
```diff
@@ -66,7 +80,10 @@ const stylingPanel = {
                       width: 9,
                       min: 5,
                       max: 300,
-                      placeholder: '',
+                      defaultValue(item, data, args) {
+                        const currentTheme = args.theme.current();
+                        return stripPx(currentTheme.object?.straightTable?.content?.fontSize || currentTheme.fontSize);
```
Comment (cbt1) [inline src/ext/property-panel/settings.js:85]:
```suggestion
                        return parseInt(currentTheme.object?.straightTable?.content?.fontSize || currentTheme.fontSize, 10);
```

**Comment by cbt1** [comment]
Looks like a zip file got commited by accident

**Comment by cbt1** [comment]
You are using the OR (||) operator but, it will give inconsistent results when the user sets a value in the theme that is truthy or falsy but not a valid font size/color. I think a safer approach is to use the nullish (??) coalesce operator instead. IMO let the user notice that a theme as invalid font size or color.

File: src/ext/property-panel/__test__/settings.spec.js [hunk 3]
```diff
@@ -61,6 +61,36 @@ describe('settings', () => {
...
 
+    describe('headerFontSize/fontSize/headerFontColor/fontColor.defaultValue', () => {
+      const { headerFontSize, headerFontColor } = headerSection.items.headerFontItem.items.headerFontWrapper.items;
+      const { contentFontSize, contentFontColor } = contentSection.items.contentFontItem.items.contentFontWrapper.items;
+      let args;
+
+      beforeEach(() => {
+        args = {
+          theme: {
+            current: () => {
```
Comment (cbt1) [inline src/ext/property-panel/__test__/settings.spec.js:72]: Is the case where you get the font-size/color from the straight table theme tested some where else? I'm not seeing it here

**Review by cbt1 — APPROVED** [review]
LGTM, maybe it lacks a test or two

**Review by LiKang6688 — APPROVED** [review]

File: src/ext/property-panel/__test__/settings.spec.js (hunk 3 above)
Comment (eliseeborn) [reply src/ext/property-panel/__test__/settings.spec.js:72]: no, I haven't tested that case, I'll add it now before merging 👍

**Review by cbt1 — APPROVED** [review]

**Pull request merged**